spec/controllers/searchgov_urls_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+describe Admin::SearchgovUrlsController do
+  fixtures :users
+
+  let(:admin) { users(:affiliate_admin) }
```

Comment:
@ethanewing , now that you're using the shared context, this line can be removed